stdlib/jdbc/src/main/java/org/ballerinax/jdbc/exceptions/DatabaseException.java

```diff
@@ -24,7 +24,7 @@
  *
  * @since 1.0.0
  */
-public class DatabaseException extends Exception {
+public class DatabaseException extends SQLException {
 
     private int sqlErrorCode;
     private String sqlState;
```

Comment:
Can't we reuse the `SQLState` and `vendorCode` properties of the `SQLException` class itself rather than defining new fields here?

Reply:
SQLState and vendorCode are defined as private variables in SQLException class

Reply:
Why can't we call the base class constructor?

Reply:
We can call
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/16926/commits/9ddfeac4d4f4489376612ed7ed2c6823eb3d94dc

Reply:
Do we need a separate  field `String reason` in `DatabaseException` class? I think we can get the same info by calling `this.getMessage()`

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/16926/commits/c30296daf3203c33243ab7441bfcb858b67aa6c7